src/apis/queryFunctions/featchApiClient.ts

```diff
@@ -87,17 +88,22 @@ async function createFetchApiClient<T>({
           return (await retryResponse.json()) as T;
         }
 
-        throw new Error('Network response was not ok');
+        const errorData = await response.json();
+
+        throw normalizeErrorKeys(errorData);
       }
 
       const errorData = await response.json();
-      throw new Error(errorData.message || 'Network response was not ok');
+
+      throw normalizeErrorKeys(errorData);
     }
 
     return (await response.json()) as T;
-  } catch (error) {
+  } catch (error: unknown) {
     console.error(error);
-    throw error;
+    // {status_code: 400, result_msg: '비밀번호가 일치하지 않습니다.', result_data: null}
+
+    throw normalizeErrorKeys(error as FetchError);
```

Comment:
어떤 에러 코드를 뱉는지 모르기 때문에 FetchError인 경우에만 FetchError 함수로 감싸는 것은 어떨까요??
```
function isFetchError(error: unknown): error is FetchError {
  return (
    typeof error === 'object' && error !== null && 'status_code' in error && 'result_msg' in error
  );
}
```

```suggestion
     if (isFetchError(error)) {
      throw normalizeErrorKeys(error);
    }

    throw error;
```